### PR TITLE
Update Brazil podcast promo with new config

### LINKED ISSUE #9588: Update Brazil podcast promo
**Is your feature request related to a problem? Please describe.**
The Brazil service want to update the podcast promo to a newer brand. 

**Describe the solution you'd like**
New config as follows

```
podcastPromo: {
      title: 'Podcast',
      brandTitle: 'BBC Lê',
      brandDescription:
        'A BBC lê para você algumas de suas melhores reportagens',
      image: {
        src: 'https://ichef.bbc.co.uk/images/ic/448xn/p09qw181.jpg',
        alt: 'BBC Lê',
      },
      linkLabel: {
        text: 'Episódios',
        href: 'https://www.bbc.com/portuguese/podcasts/p09qw1cn',
```

Update the Brazil config found here - https://github.com/bbc/simorgh/blob/latest/src/app/lib/config/services/portuguese.js

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Checklist**
- [ ] (BBC contributors only) This issue follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Additional context**

### PR DESCRIPTION
**Pull request by nicola-d-bbc**
Resolves #9588

**Overall change:**
Updates Brazil podcast promo config with newer brand.

**Code changes:**

- Updated existing podcastPromo in https://github.com/bbc/simorgh/blob/latest/src/app/lib/config/services/portuguese.js to newer brand config

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project
- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false yarn test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/lib/config/services/portuguese.js [hunk 1]
```diff
@@ -72,16 +72,16 @@ export const service = {
...
       title: 'Podcast',
-      brandTitle: 'Que História!',
+      brandTitle: 'BBC Lê',
       brandDescription:
-        'Incríveis histórias reais que nem a mente mais criativa poderia ter inventado',
+        'A BBC lê para você algumas de suas melhores reportagens',
       image: {
-        src: 'https://ichef.bbci.co.uk/images/ic/$recipe/p09dsdxd.jpg',
-        alt: 'Que História!',
+        src: 'https://ichef.bbc.co.uk/images/ic/448xn/p09qw181.jpg',
```
Comment (nicola-d-bbc) [inline src/app/lib/config/services/portuguese.js]: Not really sure what the difference is between using 'bbc' and 'bbci' in links.

**Review by mfonofm — APPROVED** [review]

File: src/app/lib/config/services/portuguese.js (hunk 1 above)
Comment (jroebu14) [reply src/app/lib/config/services/portuguese.js]:
```suggestion
        src: 'https://ichef.bbc.co.uk/images/ic/$recipe/p09qw181.jpg',
```

the `$recipe` part will get replaced with the appropriate image size in the srcset attribute here https://github.com/bbc/simorgh/blob/latest/src/app/containers/PodcastPromo/index.jsx#L18-L21

This enables us to serve different sized images to different devices.

**Review by pvaliani — APPROVED** [review]
Nice :D

**Comment by LilyL0u** [comment]
Passes testing

**Pull request merged**